### ISSUE TITLE
Multi-class MOT in QDTrack

### ISSUE THREAD

**Issue by JackWoo0831** [open-mmlab/mmdetection#11409]
Thanks for your error report and we appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
3. The bug has not been fixed in the latest version.

**Describe the bug**

When using QDTrack, I found that all the category labels of ground truths are set to 0 in `mmdet/models/mot/qdtrack.py`, func `loss` of class `QDTrack`, line `138-139`:

```python 
            key_data_sample.gt_instances.labels = \
                torch.zeros_like(key_data_sample.gt_instances.labels)
```

However, it is not suitable for training multi-class datasets like BDD100K or VisDrone. I wonder is it a bug or it refers another purpose?

**Reproduction**

1. What command or script did you run?

```none
CUDA_VISIBLE_DEVICES=3 python tools/train.py configs/qdtrack/qdtrack_visdrone_baseline.py
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?

No except my own-defined datasets.

4. What dataset did you use?

VisDrone-MOT

**Bug fix**

I ignored the codes and find everything seems okay.
